angular-hub/src/app/pages/communities/index.page.ts

```diff
@@ -50,6 +49,14 @@ export const routeMeta: RouteMeta = {
         <li>
           <app-community-card [community]="community"></app-community-card>
         </li>
+      } @empty {
+        <p class="col-span-2">
```

Comment:
minor: Why `col-span-2` instead of `col-span-full`?

Reply:
hmm, you are right, this is still WIP regarding styles but good catch man, thanks 🙌